### PR TITLE
i18n(pt-BR): Add `configuration-reference.md` translation

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

PT-BR translation for `configuration-reference.md`

### CODE REVIEW

File: src/pages/pt-BR/reference/configuration-reference.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+### publicDir
+
+<p>
+
+**Tipo:** `string`<br>
+**Padrão:** `"./public"`
+</p>
+
+Define o diretório de seus assets estáticos. Arquivos nesse diretório são servidos em `/` durante o desenvolvimento e são copiados para o diretório da sua build durante a build. Esses arquivos serão sempre servidos ou copiados da forma que são, sem transformações ou etapa de bundle.
```
Comment (caioferrarezi) [inline src/pages/pt-BR/reference/configuration-reference.md:72]: It changes a little bit, but these `build` repetition is a little bit confusing in portuguese.

```suggestion
Define o diretório de seus assets estáticos. Arquivos nesse diretório são servidos em `/` durante o desenvolvimento e são copiados para o diretório de saída durante o processo de build. Esses arquivos serão sempre servidos ou copiados da forma que são, sem transformações ou etapa de bundle.
```

File: src/pages/pt-BR/reference/configuration-reference.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+
+
+### base
+
+<p>
+
+**Tipo:** `string`
+</p>
+
+O caminho base no qual você está fazendo deploy em. Astro irá corresponder este nome de caminho durante o desenvolvimento para que sua experiência de desenvolvimento corresponda o seu ambiente de build o máximo o possível. No exemplo abaixo, `astro dev` irá iniciar seu servidor em `/docs`.
```
Comment (caioferrarezi) [inline src/pages/pt-BR/reference/configuration-reference.md:125]:
```suggestion
O caminho base no qual você está fazendo deploy. Astro irá corresponder este nome de caminho durante o desenvolvimento para que a experiência corresponda ao máximo possível o seu ambiente de build. No exemplo abaixo, `astro dev` irá iniciar seu servidor em `/docs`.
```

File: src/pages/pt-BR/reference/configuration-reference.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+
+### server.port
+
+<p>
+
+**Tipo:** `number`<br>
+**Padrão:** `3000`
+</p>
+
+Define em qual porta o servidor de desenvolvimento deve ser escutado em.
```
Comment (caioferrarezi) [inline src/pages/pt-BR/reference/configuration-reference.md:226]:
```suggestion
Define em qual porta o servidor de desenvolvimento deve ser escutado.
```

File: src/pages/pt-BR/reference/configuration-reference.md [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+
+Se a porta dada já estiver em uso, Astro irá automaticamente tentar a próxima porta disponível.
```
Comment (caioferrarezi) [inline src/pages/pt-BR/reference/configuration-reference.md:228]:
```suggestion
Se a porta indicada já estiver em uso, Astro irá automaticamente tentar a próxima porta disponível.
```

**Comment by Yan-Thomas** [comment]
Thank you @caioferrarezi!!

**Review by caioferrarezi — APPROVED** [review]
LGTM! Great work, @Yan-Thomas!

**Review by delucis — APPROVED** [review]
Thank you @Yan-Thomas & @caioferrarezi!

**Pull request merged**